modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -244,6 +244,11 @@ protected String brokerAdvertisedListener(InspectContainerResponse containerInfo
         return String.format("BROKER://%s:%s", containerInfo.getConfig().getHostName(), "9092");
     }
 
+    private static String getStarterScript() {
+        String scriptFromEnv = System.getenv("TESTCONTAINERS_KAFKA_STARTER_SCRIPT_OVERRIDE");
```

Comment:
Should I use different variables for each module? Or will the common variable `TESTCONTAINERS_STARTER_SCRIPT_OVERRIDE` be enough?